secant/Wrappers/WrappedSDKSynchronizer.swift

```diff
@@ -184,7 +188,19 @@ class LiveWrappedSDKSynchronizer: WrappedSDKSynchronizer {
         
         return .none
     }
-    
+
+    func getTransparentBalance() -> Effect<WalletBalance, Never> {
```

Comment:
see above

Reply:
yes, with 0.16.5-beta it simplified a lot of things, I refactored a lot of code. Also unit tests